app/views/task/index.html.erb

```diff
@@ -0,0 +1,40 @@
+<h1>The Queen of Task Lists</h1>
+
+<h2> Directions for Usage</h2>
+
+<ul>
+  <li>Click on the task name you wish to see more details for.</li>
+  <li>Press the delete button to delete the task.</li>
+  <li>Click the Create a Task link to make a new item on the list!</li>
+</ul>
+
+<table>
+  <thead>Honey-Do List</thead>
+  <tr>
+    <th> ID</th>
+    <th> Name</th>
+    <th> Completed?</th>
+    <th> Person Assigned </th>
+    <th> Mark as Complete </th>
+    <th> Delete </th>
+  </tr>
+
+  <tr>
+    <% @tasks.each do |task|  %>
+        <td> <%= task.id %> </td>
+         <!-- linking the task person name to the unique task id (read)-->
+    <td> <%= link_to "#{task[:name]}", "/tasks/#{task.id}/show" %></td>
```

Comment:
So Atom automatically adds the end tags for html, but puts them on a different line.  Is it alright for it to look like:
 <td
  stuff here
</td